README.md

```diff
@@ -1,38 +1,77 @@
+The following example applications can use this repository:
 
-## HAAPI Code Examples
+- [Android Kotlin AppAuth Code Example](https://curity.io/resources/learn/kotlin-android-appauth/)
+- [iOS Swift AppAuth Code Example](https://curity.io/resources/learn/swift-ios-appauth/)
+- [Advanced AppAuth code example using Dynamic Client Registration](https://curity.io/resources/learn/authenticated-dcr-example/)
+- [Android Kotlin HAAPI UI SDK Code Example](https://curity.io/resources/learn/kotlin-android-haapi/)
+- [iOS Swift HAAPI UI SDK Code Example](https://curity.io/resources/learn/swift-ios-haapi/)
+- [React Native HAAPI Code Example](https://curity.io/resources/learn/react-native-haapi/)
+- [Android Kotlin HAAPI Model SDK Code Example](https://github.com/curityio/android-haapi-demo-app)
+- [iOS Swaift HAAPI Model SDK Code Example](https://github.com/curityio/ios-haapi-demo-app)
 
-How to implement OpenID Connect using the Hypermedia Authentication API:
+## Deployment Interface
 
-| Platform | Github Repository |
-| -------- | ----------------- |
-| iOS | https://github.com/curityio/ios-haapi-demo-app |
-| Android | https://github.com/curityio/android-haapi-demo-app |
+Each mobile example calls the following script with parameters to start and stop the Curity Identity Server:
 
-## AppAuth Basic Code Examples
+```bash
+./start.sh
+./stop.sh
+```
 
-How to implement OpenID Connect using the AppAuth Pattern:
+## Configuration
 
-| Platform | Github Repository |
-| -------- | ----------------- |
-| iOS | https://github.com/curityio/openid-client-ios-appauth |
-| Android | https://github.com/curityio/openid-client-android-appauth |
+The files in the `resources` folder provide the base configuration.\
+Each code example can apply additional configuration based on its requirements:
 
-## AppAuth with Dynamic Client Registration
+- Examples can use default configuration files stored in this project, e.g. in the `appauth` folder.
+- Examples can override configuration by copying in their own configuration files, e.g to the `haapi` folder.
 
-How to implement OpenID Connect with a unique mobile client per device:
+## Administration and Users
 
-| Platform | Github Repository |
-| -------- | ----------------- |
-| iOS | https://github.com/curityio/openid-client-ios-appauth-dcr |
-| Android | https://github.com/curityio/openid-client-android-appauth-dcr |
+After deploying the Curity Identity Server, sign in to the Admin UI using the following details:
+
+- URL: `https://localhost:6749/admin`
+- User: `admin`
+- Password: `Password1`
+
+If you use ngrok and you have a suitable license file you can also sign into the DevOps Dashboard and create test user accounts:
+
+- URL: `https://localhost:6749/admin/dashboard`
+- User: `admin`
+
+You may also need to trust the `localhost` certificate at `resources/ssl-cert.pem` for the dashboard to successfully make fetch requests. For example, on macOS:
```

Comment:
It might also be worth noting that this doesn't work on FF. Firefox does not use the macOS Keychain to check the certificate trust.